packages/java-parser/src/tokens.js

```diff
@@ -105,8 +106,11 @@ const UnarySuffixOperator = createToken({
   pattern: Lexer.NA
 });
 
-// TODO: align with Java Spec
-createToken({ name: "WhiteSpace", pattern: /\s+/, group: Lexer.SKIPPED });
+createToken({
+  name: "WhiteSpace",
+  pattern: MAKE_PATTERN("[\\x09\\x20\\x0C]|{{LineTerminator}}"),
```

Comment:
Do you think it would be more readable to use specific escapes where applicable?
```javascript
var hardForMeWithHexEscapes = '[\\x09\\x20\\x0C]';
// left the hex escape for space cause otherwise it can be "missed" while reading.
var easierForMe = '[\\t\\x20\\f]'

const space = " ";
var evenEasierToRead = `[\\t${space}\\f]`;
```

Reply:
Also have a look here:
https://github.com/bd82/toml-tools/blob/master/packages/lexer/lib/tokens.js#L72

I've implemented a better DSL for composing regExps in Toml Tools Parser.
This should not be integrated as part of this change of course, but if you find this useful it could be
considered for integration in a separate PR.

Reply:
I think for readability, it is better to use specific escapes, but personnally, aesthetically, I don't like mixing up specific escapes and hex escapes.
I tried to do the "evenEasierToRead" method since I do quite like it, but it seems to make the Lexer crash.
Instead we could let the Hex escapes and comment the equivalent escape so anyone can read depending on their preferences.

Reply:
A comment next to hex escapes is fine too.